contracts/eos/BancorConverter/migrate.cpp

```diff
@@ -0,0 +1,79 @@
+[[eosio::action]]
+void BancorConverter::migrate( const set<symbol_code> converters )
+{
+    for ( const symbol_code symcode : converters ) {
+        const symbol currency = migrate_converter( symcode );
+        migrate_reserve( currency );
+    }
+}
+
+symbol BancorConverter::migrate_converter( const symbol_code symcode )
+{
+    // tables
+    converters converters_table_scope( get_self(), symcode.raw() );
+    converters converters_table_no_scope( get_self(), get_self().value );
+
+    // iterators
+    auto converter_itr_scope = converters_table_scope.find( symcode.raw() );
+    auto converter_itr_no_scope = converters_table_no_scope.find( symcode.raw() );
+
+    // scoped symbol code must exists
+    check( converter_itr_scope != converters_table_scope.end(), symcode.to_string() + " currency symbol code does not exists" );
+
+    // create
+    if ( converter_itr_no_scope == converters_table_no_scope.end() ) {
+        // add v1 converters to main scope
+        converters_table_no_scope.emplace(get_self(), [&](auto& row) {
+            row.currency = converter_itr_scope->currency;
+            row.owner = converter_itr_scope->owner;
+            row.stake_enabled = converter_itr_scope->stake_enabled;
```

Comment:
maybe we can also remove the stake and re-add it if/when it's relevant?
it's not really used today - is it a problem to re-add it later to the collection? would it require a migration to add a value?

Reply:
It would be a problem adding it in the future

We can do something "fancy" with `map<name, name>` which allows to add future features.

Also, might be useful to add `map<name, string>` for additional metadata (ex: website, logo, etc..)

Reply:
not critical, can deal with if/when needed

Reply:
added `map<name, bool> protocol_features` in new converter schema